modules/apps/jaggery/mdm/src/mdm/modules/enrollments/windows/agent-controller.jag

```diff
@@ -42,25 +43,32 @@ if (platform != "Windows Phone") {
     if (!username || !email || !password || !ownership) {
         response.sendRedirect(mdmProps["appContext"] + "enrollments/error/unintentional-request");
     } else {
-        var payload = {"credentials": {"username": username, "email": email,
-            "password": password, "ownership": ownership}};
-        var authUrl = mdmProps["windowsConfigRoot"];
-        serviceInvokers.XMLHttp.post(authUrl, payload, function (responseText) {
-            session.put("authenticatedUser", username);
-            session.put("windowsBinaryToken", responseText);
-            response.sendRedirect(mdmProps["appContext"] + "enrollments/windows/license-agent");
-        }, function (responseText) {
-            if (responseText == 403) {
+        apiWrapperUtil.setupAccessTokenPair("password", {"username": username, "password": password});
+        var tokenPair = session.get(constants.ACCESS_TOKEN_PAIR_IDENTIFIER);
+        if (tokenPair != null && tokenPair != undefined && tokenPair.accessToken != null) {
+            var payload = {"credentials": {"username": username, "email": email,
+                "password": password, "ownership": ownership, "token":tokenPair.accessToken}};
```

Comment:
already fixed it https://github.com/wso2/product-mdm/blob/master/modules/apps/jaggery/emm-web-agent/src/emm-web-agent/modules/enrollments/windows/agent-controller.jag#L57